spring-cloud-azure-context/src/main/java/com/microsoft/azure/spring/cloud/context/core/AzureAdmin.java

```diff
@@ -245,6 +248,21 @@ public RedisCache getOrCreateRedisCache(String name) {
         return getOrCreate(this::getRedisCache, this::createRedisCache, RedisCache.class).apply(name);
     }
 
+    private CosmosDBAccount getCosmosDBAccount(String name){
+        return azure.cosmosDBAccounts().getByResourceGroup(resourceGroup, name);
+    }
+
+    private CosmosDBAccount createCosmosDBAccount(String name){
+        return azure.cosmosDBAccounts().define(name).withRegion(region)
+                .withExistingResourceGroup(resourceGroup).withKind(DatabaseAccountKind.MONGO_DB)
+                .withStrongConsistency().withReadReplication(Region.US_EAST).create();
```

Comment:
may need to consider other `DatabaseAccountKind` instead of hard code MONGO here.

Reply:
Agree. You need to make db kind as parameter.